pages/common/ansible-galaxy.md

```diff
@@ -22,3 +22,7 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get the information for the role - or a collection if you replace `role` with `collection`:
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Get the information for the role - or a collection if `role` is replaced with `collection`:
```